gammapy/modeling/models/temporal.py

```diff
@@ -11,14 +11,94 @@
 from gammapy.utils.scripts import make_path
 from gammapy.utils.time import time_ref_from_dict
 
-__all__ = ["PhaseCurveTemplateTemporalModel", "LightCurveTemplateTemporalModel", "TemporalModel"]
+__all__ = [
+    "PhaseCurveTemplateTemporalModel",
+    "LightCurveTemplateTemporalModel",
+    "TemporalModel",
+    "ConstantTemporalModel",
+]
 
 
 # TODO: make this a small ABC to define a uniform interface.
 class TemporalModel(Model):
     """Temporal model base class"""
 
 
+class ConstantTemporalModel(TemporalModel):
+    """Constant temporal model.
+
+    Parameters
+    ----------
+    norm : float
+        The normalization of the constant temporal model
+
+    Examples
+    --------
+    Create an example constant lightcurve::
+    >>> time = np.arange(0, 1, 0.1)
+    >>> norm = 10.0
+    >>> const_mod = ConstantTemporalModel(norm)
+    >>> const_mod.evaluate_norm_at_time(time)
+    array([10., 10., 10., 10., 10., 10., 10., 10., 10., 10.])
+    """
+
+    def __init__(self, norm):
+        self.norm = Parameter("norm", norm)
+
+    def evaluate_norm_at_time(self, time):
+        """Evaluate for a given time.
+
+        Parameters
+        ----------
+        time : array_like
+            Time since the ``reference`` time.
+
+        Returns
+        -------
+        norm : float
+            Mean norm
+        """
+
+        return np.ones(len(time), float) * self.norm.value
+
+    def sample_time(self, n_events, t_min, t_max, t_delta="1 s", random_state=0):
```

Comment:
Note the `t_delta` is not needed anymore, please remove...

Reply:
OK.